back/lib/participation_method/native_survey.rb

```diff
@@ -127,8 +127,36 @@ def supports_toxicity_detection?
       false
     end
 
+    # Attribute used interally by backend to determine if user fields should be shown in the form
```

Comment:
Kind of weird to keep this attribute (and the `user_data_collection` as attributes of this participation method object. But I was not sure where else to put it, and it is kind of convenient since we usually use these attributes in the BE in places where we already have access to the phase.